tests/user_examples/extra_src/shared/general_continuum/continuum_particles.h

```diff
@@ -27,13 +24,13 @@ namespace SPH
         StdLargeVec<Vecd> n_;    /**<  current normal direction */
         StdLargeVec<Vecd> n0_;   /**<  initial normal direction */
 
-        GeneralContinuum &continuum_;
+        GeneralContinuum& continuum_;
 
-        ContinuumParticles(SPHBody &sph_body, GeneralContinuum *continuum);
-        virtual ~ContinuumParticles(){};
+        ContinuumParticles(SPHBody& sph_body, GeneralContinuum* continuum);
```

Comment:
Here, why not directly pass continuum as reference?

Reply:
This is what we do for the `BaseParticles`, and `ContinuumParticles` is inherited from `BaseParticles`.